engines/python/setup/djl_python/rolling_batch/scheduler_rolling_batch.py

```diff
@@ -111,17 +113,30 @@ def _init_model_and_tokenizer(self,
             if 'device_map' in kwargs:
                 device_map = kwargs.pop('device_map')
 
-            self.model = AutoModelForCausalLM.from_pretrained(
-                model_id_or_path, device_map=device_map, **kwargs)
+            if 'neox' in model_id_or_path and int(
```

Comment:
you can just grab the model from lmi_dist init function right?

Reply:
could we add some special flags to use LMI dist library? Because sometimes we need fallback option of using HF accelerate